addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -828,13 +828,16 @@ body.editor_enable.editor_has_snippets {
                 }
             }
             .o_we_pager_next, .o_we_pager_prev {
+                margin: 3px;
                 padding: $o-we-sidebar-content-field-label-spacing;
                 cursor: pointer;
+                border: 1px solid;
+                border-radius: 2px;
             }
             we-select-page {
                 display: none;
                 width: 100%;
-                max-height: 600px;
+                max-height: 75px * 7.5;
```

Comment:
Add a comment to explain why that 7.5 value (I understand by reading the commit but I should not have to blame the code if I stumble upon this line later on).